sql/telemetry/rocket_android_events_v1/view.sql

```diff
@@ -155,32 +153,42 @@ SELECT
+        WHEN (event_category IN ('action') ) AND (event_method IN ('show') ) AND (event_object IN ('search_bar') ) AND (event_value IN ('content_home') ) THEN 'Rocket -  Show Content Home Search Bar' 
+        WHEN (event_category IN ('action') ) AND (event_method IN ('select_query') ) AND (event_object IN ('search_bar') ) AND (event_value IN ('content_home') ) THEN 'Rocket -  Select Query Content Home' 
         WHEN (event_category IN ('action') ) AND (event_method IN ('open') ) AND (event_object IN ('category') ) THEN 'Rocket -  Open Category' 
+        WHEN (event_category IN ('action') ) AND (event_method IN ('open') ) AND (event_object IN ('detail_page') ) AND (event_value IN ('more') ) THEN 'Rocket -  Open Detail Page More' 
         WHEN (event_category IN ('action') ) AND (event_method IN ('start') ) AND (event_object IN ('content_tab') ) THEN 'Rocket -  Start Content Tab' 
         WHEN (event_category IN ('action') ) AND (event_method IN ('end') ) AND (event_object IN ('content_tab') ) THEN 'Rocket -  End Content Tab' 
         WHEN (event_category IN ('action') ) AND (event_method IN ('impression') ) AND (event_object IN ('category') ) THEN 'Rocket -  Category Impression' 
         WHEN (event_category IN ('action') ) AND (event_method IN ('start') ) AND (event_object IN ('process') ) AND (event_value IN ('vertical') ) THEN 'Rocket -  Start Vertical Process' 
         WHEN (event_category IN ('action') ) AND (event_method IN ('end') ) AND (event_object IN ('process') ) AND (event_value IN ('vertical') ) THEN 'Rocket -  End Vertical Process' 
+        WHEN (event_category IN ('action') ) AND (event_method IN ('show') ) AND (event_object IN ('onboarding') ) AND (event_value IN ('vertical') ) THEN 'Rocket -  Show Vertical Onboarding' 
         WHEN (event_category IN ('action') ) AND (event_method IN ('start') ) AND (event_object IN ('process') ) AND (event_value IN ('tab_swipe') ) THEN 'Rocket -  Start Tab Swipe Process' 
         WHEN (event_category IN ('action') ) AND (event_method IN ('end') ) AND (event_object IN ('process') ) AND (event_value IN ('tab_swipe') ) THEN 'Rocket -  End Tab Swipe Process' 
         WHEN (event_category IN ('action') ) AND (event_method IN ('start') ) AND (event_object IN ('tab_swipe') ) THEN 'Rocket -  Start Tab Swipe' 
         WHEN (event_category IN ('action') ) AND (event_method IN ('end') ) AND (event_object IN ('tab_swipe') ) THEN 'Rocket -  End Tab Swipe' 
-        WHEN (event_category IN ('action') ) AND (event_method IN ('click') ) AND (event_object IN ('toolbar') ) AND (event_value IN ('share|OPEN_IN_BROWSER|reload|back') ) THEN 'Rocket -  Click Toolbar - Share/Reload/Back' 
+        WHEN (event_category IN ('action') ) AND (event_method IN ('click') ) AND (event_object IN ('toolbar') ) AND (event_value IN ('share', 'OPEN_IN_BROWSER', 'reload', 'back') ) THEN 'Rocket -  Click Toolbar - Share/Reload/Back' 
```

Comment:
not aligned with spec
https://docs.google.com/spreadsheets/d/1fralUDT5tZWZgfyFvAhbxHK7b1UQz696yL0xPrdu45A/edit#gid=514507148&range=C12

should be
`WHEN (event_category IN ('action') ) AND (event_method IN ('click') ) AND (event_object IN ('toolbar') ) AND (event_value IN ('share')) THEN 'Rocket -  Click Toolbar - Share' `

`WHEN (event_category IN ('action') ) AND (event_method IN ('click') ) AND (event_object IN ('toolbar') ) AND (event_value IN ('open_in_browser') ) THEN 'Rocket -  Click Toolbar - Open in Browser' `

`WHEN (event_category IN ('action') ) AND (event_method IN ('click') ) AND (event_object IN ('toolbar') ) AND (event_value IN ('reload' ) ) THEN 'Rocket -  Click Toolbar - Reload' `

`WHEN (event_category IN ('action') ) AND (event_method IN ('click') ) AND (event_object IN ('toolbar') ) AND (event_value IN ('back' ) ) THEN 'Rocket -  Click Toolbar - Back' `


please kindly check again

Reply:
@cnevinc